bnb_yaz/bnb-downloader.py

```diff
@@ -147,8 +147,27 @@ def DownloadRecordsRanges(yaz_client, ranges):
     return download_count
 
 
+def SetupWorkDirectory():
+    work_directory: str = "/tmp/bnb-downloader.work"
+    try:
+        os.mkdir(work_directory, mode=0o744)
+    except:
+        pass
+    os.chdir(work_directory)
+
+    
+def UploadToBSZFTPServer(marc_filename: str):
+    remote_file_name_tmp: str = marc_filename + ".tmp"
+
+    ftp = bsz_util.GetFTPConnection()
+    ftp.changeDirectory(remote_folder_path)
+    ftp.uploadFile(marc_filename, remote_file_name_tmp)
+    ftp.renameFile(remote_file_name_tmp, marc_filename)
+
+
 def Main():
     util.default_email_recipient = "johannes.ruscheinski@uni-tuebingen.de"
```

Comment:
Maybe our ixtheo-team-Address?

Reply:
I still consider this to be in testing and will eventually simply delete this line.